core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java

```diff
@@ -41,24 +41,24 @@
     private boolean enableCoordinatorDynamicFiltersDistribution = true;
     private boolean enableLargeDynamicFilters;
 
-    private int smallBroadcastMaxDistinctValuesPerDriver = 200;
-    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(20, KILOBYTE);
-    private int smallBroadcastRangeRowLimitPerDriver = 400;
-    private DataSize smallBroadcastMaxSizePerOperator = DataSize.of(200, KILOBYTE);
-    private int smallPartitionedMaxDistinctValuesPerDriver = 20;
-    private DataSize smallPartitionedMaxSizePerDriver = DataSize.of(10, KILOBYTE);
-    private int smallPartitionedRangeRowLimitPerDriver = 100;
-    private DataSize smallPartitionedMaxSizePerOperator = DataSize.of(100, KILOBYTE);
-    private DataSize smallMaxSizePerFilter = DataSize.of(1, MEGABYTE);
-
-    private int largeBroadcastMaxDistinctValuesPerDriver = 5_000;
-    private DataSize largeBroadcastMaxSizePerDriver = DataSize.of(500, KILOBYTE);
-    private int largeBroadcastRangeRowLimitPerDriver = 10_000;
+    private int smallBroadcastMaxDistinctValuesPerDriver = 1_000;
+    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(200, KILOBYTE);
```

Comment:
why not 100kb (as multiplying everything by 5)?

Reply:
will update to 100kb for consistency